projects/gnomad/src/client/VariantPage/TranscriptConsequenceDetails.js

```diff
@@ -1,8 +1,40 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import styled from 'styled-components'
 
 import { getCategoryFromConsequence } from '@broad/utilities'
 
+const AttributeName = styled.dt`
+  display: inline;
+
+  ::after {
+    content: ': ';
+  }
+`
+
+const AttributeValue = styled.dd`
+  display: inline;
+  margin: 0;
+`
+
+const AttributeList = styled.dl`
```

Comment:
Should we do validation on AttributeList's children, a la https://stackoverflow.com/questions/27366077/only-allow-children-of-a-specific-type-in-a-react-component/36424582#36424582?

Reply:
I had not considered validating children/props of styled components, but that makes sense for cases where there's some expected nesting.